keras_cv/bounding_box/utils.py

```diff
@@ -77,6 +78,7 @@ def clip_to_image(bounding_boxes, images, bounding_box_format):
         source=bounding_box_format,
         target="rel_xyxy",
         images=images,
+        image_shape=image_shape,
```

Comment:
can't we get the image shape from images?

Reply:
no we can't in the utility users may provide one OR the other, but not both.  image_shape is more performant but requires all images to be of the same shape (ie no raggeds)